packages/widgets-manager/src/grid.spec.ts

```diff
@@ -133,8 +143,10 @@ describe('GridManager', () => {
       id: 'center-center',
       apps: [mockApp1],
       position: {
-        x: windowObject.innerWidth / 3,
-        y: windowObject.innerHeight / 3,
+        left: windowObject.innerWidth / 3,
```

Comment:
Where does this **3** and **9**(in mobile case) come from ?

Reply:
This is a 3x3 grid (in desktop) so the element at center-center is (from the left) at 1/3 window width.
In mobile we have a 9x1 (only one column, items are vertical) so the top-center element (the second element to appear) should be at (window Height / 9) 
This is because we explicitly set that in mobile we have just 1 column.